XRTK-Core/Packages/com.xrtk.core/Definitions/Devices/IMixedRealityHandJointService.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using XRTK.Definitions.Utilities;
+
+namespace XRTK.Definitions.Devices
+{
+    /// <summary>
+    /// Mixed Reality Toolkit device definition, used to instantiate and manage a specific device / SDK
+    /// </summary>
+    public interface IMixedRealityHandJointService : IMixedRealityInputDeviceManager
```

Comment:
`IMixedRealityInputDeviceManager` not avaialble in XRTK. Not sure how to resolve yet.

Reply:
Lets review to see if it's applicable

Reply:
Resolved using `IMixedRealityControllerDataProvider`